client/cypress/integration/user/edit_profile_spec.js

```diff
@@ -36,14 +41,6 @@ describe('Edit Profile', () => {
     });
```

Comment:
IMO setting the state after the test is not a good approach as well. My ideas to improve it are:
- try to intercept API response and modify "api_key" from it (it was not as easy as I thought to do it ^^);
- modify the `edit_profile_spec` to do something I should've done at first, but leaned later: create a user to be tested prior the tests instead of asserting on the Admin user (this should fix this one).

The goal here is to get some time to do the above without annoyance in Percy. LMK what you think :slightly_smiling_face:

Reply:
The best approach IMO is to actually "blackout" the element so it doesn't get included in screenshots.
```css
@media only percy {
  .hide-in-percy {
    visibility: hidden;
  }
}
```

Reply:
@gabrieldutra if you go this route, can you piggyback [pace](https://github.com/getredash/pace) as well (it's causing Percy false negatives)

Reply:
Not sure this is the best solution to those cases as we are not capturing changes to that part of the UI. However this seems the best for now and I wanted to introduce Percy specific css anyway :slightly_smiling_face:.

@ranbena please leave your comments, when you think this is ok I'll rename the PR and merge it.